src/lib/scheduler/operator_task.cpp

```diff
@@ -60,6 +98,16 @@ OperatorTask::make_tasks_from_operator(const std::shared_ptr<AbstractOperator>&
   auto operator_tasks_set = std::unordered_set<std::shared_ptr<OperatorTask>>{};
   const auto& root_operator_task = add_operator_tasks_recursively(op, operator_tasks_set);
 
+  // Predicates that contain uncorrelated subqueries cannot be used for chunk pruning in the optimization phase since we
+  // do not know the predicate value yet. However, the ChunkPruningRule attaches the corresponding PredicateNodes to the
+  // StoreTableNode of the table the predicates are performed on. We attach the translated Predicates (i.e., TableScans)
+  // to the GetTable operators so they can use them for pruning during execution. We set the tasks associated with the
+  // unccorrelated subqueries as predecessors of the GetTable tasks so the GetTable operators can extract the predicate
+  // values and perform dynamic chunk pruning. We cannot link the tasks during the recursive creation of operator tasks.
+  // Since not all tasks might be created when reaching the GetTable tasks, we cannot traverse the tasks to ensure an
```

Comment:
plase form more than a sentence fragment 😅

Reply:
What does "not all tasks might be created" mean? Some tasks are never created?
Do you mean, that they might not YET exist?

Reply:
Might be my bad English, but I think it can be misread. Maybe just a "yet"?  `Potentially, not all tasks are yet created when reaching the GetTable tasks"?

Reply:
Don't you know somebody who speaks English to ask?

Reply:
I'm from Eastern Germany, how should I know someone who speaks English? o.O